assets/dev/scss/frontend/_container.scss

```diff
@@ -44,9 +58,12 @@
 	min-height: var(--min-height);
 	height: var(--height);
 	border-radius: var(--border-radius);
-	margin: var(--margin-top, var(--margin-block-start)) var(--margin-right, var(--margin-inline-end)) var(--margin-bottom, var(--margin-block-end)) var(--margin-left, var(--margin-inline-start));
-	padding-inline-start: var(--padding-left, var(--padding-inline-start));
-	padding-inline-end: var(--padding-right, var(--padding-inline-end));
+	margin-block-start: var(--margin-top, var(--margin-block-start));
```

Comment:
Glad to see some logical shorthand joining the ranks on margin here.

Reply:
@Barcov I want to override the old/BC CSS variable '--margin-top'.
The only reason why I renamed the inline variables, is so that I could distinguish more clearly between left and right.

I could also include the block variables for consistency and clarity. Would you prefer that?